exchange/exchangetest/multi-bids-with-extra-bids.json

```diff
@@ -0,0 +1,489 @@
+{
```

Comment:
Could you add another JSON exchange test where the number of bids returned by a seat for a particular impID is greater than the maxBids value associated with that seat? That way we can see that bids are properly including/excluding targeting data.

Reply:
`multi-bids.json` handles this case where seat `pubmatic` gets 3 bids for impID `my-imp-id` and `maxBids` in request is 2. Hence, in the response we have targeting applied only for the first 2 bids and the last bid with id `losing-bid` does not have any targeting.

Am I missing anything here?